README.md

```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
 
 ##### Running a Single Test
 
-If you haven't already done so, you will need to configure the HCA CLI tool to use the EBI installation of the upload service rather than the DCP (default) version.
+To run a single test, make sure that all necessary environment variables are provided.
 
 ```
-./setup_ingest_config.sh
-```
-
-To run a single test, make sure that all necessary environment variables are provided. At the time of writing, the most
-commonly required variables are `DEPLOYMENT_ENV` and `GOOGLE_APPLICATION_CREDENTIALS`.
-
-```
-export AWS_PROFILE=embl-ebi \
+export AWS_PROFILE=embl-ebi; \
```

Comment:
I think we should start using `.env` files.

Reply:
maybe.
@amnonkhen Could you create a ticket for that one, please?